journald: add examples to show user journal logs (#297)
flatcar-linux/flatcar-docs: docs/setup/debug/reading-the-system-log.md

Dec 22 12:32:39 localhost docker[9772]: apache2: Could not reliably determine the server's fully qualified domain name, using 172.17.0.6 for ServerName
```

## Read the user journal from the current user

> It might be required to provide a user different from `core` user and in the `systemd-journal` group to read user journal. It can be done with Butane config:

Reviewer: ```suggestion
It might be required to add a user different from `core` user to the `systemd-journal` group to read the user journal. It can be done with this Butane config:
```

```yaml
variant: flatcar
version: 1.0.0
passwd:
  users:
    - name: flatcar
storage:
  files:

>     - path: /etc/userdb/flatcar:systemd-journal.membership

Reviewer: (Same here)
